source/tutorials/sample-classifier.rst

```diff
@@ -30,9 +30,9 @@ Supervised learning classifiers predict the categorical metadata classes of unla
 
 Next, we will train and test a classifier that predicts which body site a sample originated from based on its microbial composition. We will do so using the ``classify-samples`` pipeline, which performs a series of steps under the hood:
 
-1. The input samples are randomly split into a ``training`` set and a ``test`` set. The test set is held out until the end of the pipeline, allowing us to test accuracy on a set of samples that was not used for model training. The fraction of test samples to remove is adjusted with the ``--p-test-size`` parameter.
+1. The input samples are randomly split into a ``training`` set and a ``test`` set. The test set is held out until the end of the pipeline, allowing us to test accuracy on a set of samples that was not used for model training. The fraction of input samples to include in the test set is adjusted with the ``--p-test-size`` parameter.
 
-2. We train the learning model using the training set samples. The model is trained to predict a specific ``target`` value for each sample (contained in a metadata column) based on the feature data associated with that sample.
+2. We train the learning model using the training set samples. The model is trained to predict a specific ``target`` value for each sample (contained in a metadata column) based on the feature data associated with that sample. A range of different learning models can be selected using the ``estimator`` parameter; more details on individual estimators can be found in the `scikit-learn documentation`_.
```

Comment:
I personally think of "models" as being what we develop when we've got some mechanistic idea for how the target relates to the inputs. But a quick google search reveals that "machine learning model" is a common term so I'll just leave this as a comment.

Reply:
Thanks @BenKaehler ! Yeah I think there are 2 (or more) different terminology camps.